extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/output/RetryableS3OutputStream.java

```diff
@@ -212,11 +212,10 @@ public void close() throws IOException
     // Closeables are closed in LIFO order
     closer.register(() -> {
       org.apache.commons.io.FileUtils.forceDelete(chunkStorePath);
-      LOG.info("Deleted chunkStorePath[%s]", chunkStorePath);
 
       final long totalBytesUploaded = (currentChunk.id - 1) * chunkSize + currentChunk.length();
       final long totalUploadTimeMillis = pushStopwatch.elapsed(TimeUnit.MILLISECONDS);
-      LOG.info(
+      LOG.debug(
```

Comment:
can you call this method in `log.isDebugEnabled` instead? Computing the log message is expensive and we don't wanna do that if the logging level isn't `DEBUG`

Reply:
Done

Reply:
Nit: The log message is formatted only if the logging is needed.

https://github.com/apache/druid/blob/84192b11d72ed195c42827a2e142911f3f87e6ea/processing/src/main/java/org/apache/druid/java/util/common/logger/Logger.java#L128-L133

Checking `isDebugEnabled()` is useful in the Druid code only when one of the args passed to the log itself requires some computational effort. Example:

```
if (log.isDebugEnabled()) {
  log.debug("Log the result of a computation[%s].", someComputation());
}
```

Reply:
I see, thanks for mentioning this.

Reply:
@kfaraz Thanks for pointing this out, have made the change!